wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -33,7 +33,7 @@ Listed below are the properly supported locales for the wiki:
 | `no.md` | Norwegian | Norsk |
 | `pl.md` | Polish | Polski |
 | `pt.md` | Portuguese | Português |
-| `pt-br.md` | Brazilian Portuguese | Português (Brasil) |
+| `pt-br.md` | Portuguese (Brazil) | Português (Brasil) |
```

Comment:
aren't these supposed to be just how the language is called in the local language? for example wikipedia calls this "Brazilian Portuguese" https://en.wikipedia.org/wiki/Brazilian_Portuguese

Reply:
just like portuguese wikipedia, it calls it "Português brasileiro" <https://pt.wikipedia.org/wiki/Português_brasileiro>

Reply:
i see even that page doesn't use parentheses. i don't have a strong opinion. @TicClick do you?

Reply:
I'd go without parenthesis

Reply:
The parentheses are because on the osu! web it is in parentheses

Reply:
column 3 is copied from osu-web, but column 2 can have whatever formatting feels natural for the language of the article

for the english article, in my opinion your change to pt-br should be reverted but the change to zh is okay. I probably just prefer to not have extra parenthesis in there

for other language articles, please don't update them if you don't speak the language... I guess it's okay to keep the changes that have been approved by their respective translators though

Reply:
done in https://github.com/ppy/osu-wiki/pull/9964/commits/ab0c6b4765d260daa0679932aec4f92d8b58692d